tests/php/ORM/SQLSelectTest.php

```diff
@@ -1327,4 +1327,98 @@ public function testMultipleWithDuplicateName()
 
         $select->addWith('cte', new SQLSelect());
     }
+
+    public function subqueryProvider()
+    {
+        return [
+            'no-alias-string' => ['( SELECT DISTINCT "SQLSelectTest_DO"."ClassName" FROM "SQLSelectTest_DO") AS "FINAL"'],
```

Comment:
That's the only scenario that would have failed prior to this PR. The other ones are just there for the sake of completeness.